modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosSouthController.scala

```diff
@@ -85,7 +85,7 @@ object GmosSouthController {
         case (BuiltinROI.CUSTOM, r) if r.nonEmpty => \/.right(new RegionsOfInterest(\/.right(r)) {})
         case _ => \/.left(Unexpected("Inconsistent values for GMOS regions of interest"))
       }
-    def unapply(r: RegionsOfInterest): Option[BuiltinROI \/ List[ROI]] = Some(r.rois)
+    def unapply(r: RegionsOfInterest): Some[BuiltinROI \/ List[ROI]] = Some(r.rois)
```

Comment:
I'm not saying it's wrong, maybe using the constructor in the type signature in `unapply` makes more sense. I was just curious to know why it was in the type signature.